DataDog/documentation
# Add app signup geo logic

### What does this PR do?

This adds geolocation logic to the free trial CTAs so the Region selector in the signup iframe defaults to the appropriate region based on the user's location e.g. if you click the CTA while in the EU, the region will be pre-selected as `Europe (EU1)`. This functionality already exists on corpsite. The default will be US5 based on the config [here](https://github.com/DataDog/marketing-lambdas/blob/master/cloudfront-functions/geo-signup-corp.js#L36)

The babel config was updated to account for the `node_module` import that is now being done in the `geo.test.js` file.


### Motivation
https://datadoghq.atlassian.net/browse/WEB-3082

### How to test
https://docs-staging.datadoghq.com/ccole/app-signup-geo-logic/

1. Clear local storage
2. Click the `Get Started Free` or `Free Trial` CTA and verify the proper region is selected.

### Merge instructions



### Additional notes
